tests/test_ilastik/test_applets/pixelClassification/testPixelClassificationGui.py

```diff
@@ -266,8 +267,8 @@ def impl():
             selections = numpy.array(
                 [
                     [True, False, False, False, False, False, False],
-                    [True, False, False, False, False, False, False],
-                    [True, False, False, False, False, False, False],
+                    [False, True, False, False, False, False, False],
+                    [False, True, False, False, False, False, False],
```

Comment:
previous feature set was "illegal" for ilastik. It is not possible to achieve using the gui.

But main reason for changing it is that the SuggestFeatures Dialog that later is used in this test, makes this assumption